source/auth/auth.rst

```diff
@@ -1093,6 +1087,17 @@ Drivers may need to remove support for association of more than one credential w
 	* Deprecation and removal of MongoClient constructors that take as an argument more than a single credential
 	* Deprecation and removal of methods that allow lazy authentication (i.e post-MongoClient construction)
 
+Drivers need to support both the shorter and longer SCRAM conversations over mongodb's sasl implementation. Earlier versions of the server required an extra round trip due to an implementation decision. This was accomplished by sending no bytes back to the server, as seen in the following conversation (extra round trip italicized):
+
+| C: :javascript:`{saslStart: 1, mechanism: "SCRAM-SHA-1", payload: BinData(0, "biwsbj11c2VyLHI9ZnlrbytkMmxiYkZnT05Sdjlxa3hkYXdM")}`
```

Comment:
This should also contain `options: { skipEmptyExchange: true }`.  Older server versions will just ignore it iirc.

Reply:
Done